frontend/webapp/static/js/custom/actions.js

```diff
@@ -0,0 +1,35 @@
+function postLearnConfig(url, obj) { // eslint-disable-line no-unused-vars
+    return new Promise(result => {
+        fetch(url, {
+            headers: {
+              'Accept': 'application/json',
+              'Content-Type': 'application/json'
+            },
+            method: "POST",
+            body: JSON.stringify(obj)
+        })
+        .then(response => response.json())
+        .then(data => {
+                result(data);
+            }
+        )
+    });
+}
+
+function postIgnoreButton(url, obj) { // eslint-disable-line no-unused-vars
```

Comment:
the functions postLearn Config and postIgnoreButton are exactly the same, better use a single function named e.g., fetch_json (we can reuse it in multiple parts of the code).

Reply:
done